packages/react/src/components/Address/index.tsx

```diff
@@ -13,12 +12,10 @@ export const Address = ({ children, className, ...other }: AddressProps) => (
 );
 
 Address.displayName = 'Address';
-Address.propTypes = {
-  children: PropTypes.node.isRequired,
-  className: PropTypes.string
-};
 
-type AddressLineProps = React.HTMLAttributes<HTMLElement>;
+type AddressLineProps = React.HTMLAttributes<HTMLElement> & {
+  className?: string;
+};
```

Comment:
Isn't the `className` already implied?

Reply:
It is, but for some reason `react/prop-types` isn't picking it up, so it's throwing all kinds of linter errors where it was previous validating `className` via `propTypes`.

So the options are either A) ignore `className` with some linter config, or B) ensure that `className` is included as part of props.

Reply:
Ah, I see. It's because `className` isn't an `HTMLAttribute` of `HTMLElement`. We could instead potentially do `React.HTMLProps` or `React.ComponentProps` if our version of React supports it. See: https://stackoverflow.com/questions/48198180/what-is-the-difference-between-react-htmlprops-and-react-htmlattributest

Reply:
I'm down for that change, but I think that work should be handled separately from this since it does seems like there's some differences between `ComponentProps` and `HTMLAttributes` that we should try to document first before diving in.

Reply:
https://github.com/dequelabs/cauldron/issues/1252